Runtime/Scripts/Export/StandardMaterialExport.cs

```diff
@@ -104,9 +103,16 @@ public override bool ConvertMaterial(UnityEngine.Material uMaterial, out Materia
 #if UNITY_EDITOR
                         // Convert texture type to Texture2D
                         string url = UnityEditor.AssetDatabase.GetAssetPath(uMaterial.GetTexture(k_Tex));
-                        Texture2D tex = new Texture2D(2, 2);
-                        tex.LoadImage(System.IO.File.ReadAllBytes(url));
-                        uMaterial.mainTexture = tex;
+                        Texture2D texture = new Texture2D(2, 2);
+                        texture.LoadImage(System.IO.File.ReadAllBytes(url));
+
+                        int maxSize = 8192; // Firefox size limit
```

Comment:
We could use a constant for this number.

Reply:
Yes, that makes sense. Would like Daniel's input on the mobile browser size limit as well to choose a good number.

Reply:
I think we can also do
```csharp
var importer = (UnityEditor.TextureImporter)UnityEditor.AssetImporter.GetAtPath(url);
int maxSize = importer.maxTextureSize;
```

Reply:
This depends on your Unity platform texture settings, I think. For me the defaults seem to be 2048. I guess that should be safe but let me see if I can get information on the mobile size limits.

Reply:
Seems like 2048 covers most mobile devices. 4096 would still cover a majority but let's play it safe.

Reply:
Skybox is visible on chromium (specifically brave browser) on android with the new change, but a maxSize of 8192 doesn't work.